### ISSUE TITLE
The language and docs version switcher is impossible to use with a keyboard

### ISSUE THREAD

**Issue by sarahboyce** [django/djangoproject.com#1955]
https://docs.djangoproject.com/en/5.1/

You cannot tab to focus on the language and documentation version widget.
This means the options, which are only visible on hover, are not available to keyboard users.
